src/pocketmine/Player.php

```diff
@@ -3279,8 +3294,19 @@ public function addTitle(string $title, string $subtitle = "", int $fadeIn = -1,
 	 * Sets the subtitle message, without sending a title.
 	 *
 	 * @param string $subtitle
+     *
+     * @deprecated please use sendSubTitle
```

Comment:
We prefer the style
```php
@deprecated
@see Player::sendSubTitle()
```
so that it can be linked by the documentation generator. Also, please move this above the @param decls.